generators/app/templates/src/app/components/Checkbox/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+interface Props {
+  checked: boolean;
+  className?: string;
+  disabled?: boolean;
+  error?: string;
+  errorClassName?: string;
+  inputClassName?: string;
+  label?: string;
+  labelClassName?: string;
+  name: string;
+  onChange: (event: React.FormEvent<HTMLInputElement>) => void;
+  required?: boolean;
+}
+
+function Checkbox({
+  checked,
+  className = '',
+  disabled = false,
+  error = '',
+  errorClassName = '',
+  inputClassName = '',
+  label = '',
+  labelClassName = '',
+  name,
+  onChange,
+  required = false
+}: Props) {
+  return (
+    <div className={`column start ${className}`}>
+      <div>
```

Comment:
Do we need this `div`?

Reply:
I need to put `input` and `label` in the same row and the error message below